data/api.ts

```diff
@@ -203,16 +203,17 @@ export async function deleteReplicationProfile(provider: string, datasetID: numb
 }
 
 export interface GetReplicationsConfig {
-  offset: number;
-  limit: number;
-  datasets: string[];
-  providers: string[];
-  timeStart: Date;
-  timeEnd: Date;
+  offset: number,
+  limit: number,
+  datasets: string[],
+  providers: string[],
+  timeStart: Date,
+  timeEnd: Date,
+  statuses: string[],
   selfService: boolean,
   proposalCID: string;
```

Comment:
why does this one have `;` but the rest have commas?

Reply:
LOL oops thanks for noticing, they were all with semicolons before for some reason and i was cleaning them up and i missed that one